cpu/sam0_common/periph/usbdev.c

```diff
@@ -584,12 +584,23 @@ static inline void _enable_ep_stall_in(UsbDeviceEndpoint *ep_reg)
 
 static inline void _disable_ep_stall_out(UsbDeviceEndpoint *ep_reg)
 {
-    ep_reg->EPSTATUSCLR.reg = USB_DEVICE_EPSTATUSCLR_STALLRQ0;
+    ep_reg->EPSTATUSCLR.reg = USB_DEVICE_EPSTATUSCLR_STALLRQ0 |
+                              USB_DEVICE_EPSTATUSCLR_DTGLOUT;
```

Comment:
This change and the same below cause the problem. When I change it back, it works again.